images/run-playbook/README.md

```diff
@@ -0,0 +1,43 @@
+# Containerized openshift-ansible to run generic playbooks
+
+This image uses the [playbook2image](https://github.com/aweiteka/playbook2image) source-to-image image to containerize `openshift-ansible`. The resulting image can run any of the provided playbooks.
+
+## Build
+
+1. **Docker**:
```

Comment:
@aweiteka The main use case is being able to run the installer on one of the hosts that is part of the installation.

The issue we have today is that as part of the installation we are making changes to the docker config which require a service restart. If the installer is running in a container under docker, then it goes *poof* when the docker restart happens.